openmetadata-ui/src/main/resources/ui/src/components/AddWebhook/WebhookConstants.tsx

```diff
@@ -11,6 +11,9 @@
  *  limitations under the License.
  */
 
+import { startCase } from 'lodash';
+import { Status } from '../../generated/entity/events/webhook';
+
```

Comment:
Can we change the extension of this file from `.tsx` to `.ts` as it's only returning the constants?